benchmarks/src/jmh/java/software/amazon/jdbc/benchmarks/PluginBenchmarks.java

```diff
@@ -75,7 +77,8 @@ public class PluginBenchmarks {
   private static final String PG_CONNECTION_STRING =
       "jdbc:aws-wrapper:postgresql://instance-0.XYZ.us-east-2.rds.amazonaws.com";
   private static final int TEST_PORT = 5432;
-  private final HostSpec writerHostSpec = new HostSpec("instance-0", TEST_PORT);
+  private final HostSpec writerHostSpec = new HostSpecBuilder(new SimpleHostAvailabilityStrategy())
+      .host("instance-0").port(TEST_PORT).build();
```

Comment:
Can we make `instance-0` a variable too for consistency?

Reply:
Done!